Makefile

```diff
@@ -136,28 +138,28 @@ run-backend-debug: ## Runs all backend containers in debug mode, where all conta
 
 ## ------------------------------------------------------------------------------
 ## Build/Run Backend+Web Commands (backend + web frontend)
-## - Builds all of the biohub backend+web projects (db, db_setup, api, app, n8n, n8n_nginx, n8n_setup)
+## - Builds all of the biohub backend+web projects (db, db_setup, api, app, n8n, n8n_nginx, n8n_setup clamav)
 ## ------------------------------------------------------------------------------
 
 build-web: ## Builds all backend+web containers
 	@echo "==============================================="
 	@echo "Make: build-web - building web images"
 	@echo "==============================================="
-	@docker-compose -f docker-compose.yml build db db_setup api app n8n n8n_nginx n8n_setup
+	@docker-compose -f docker-compose.yml build db db_setup api app n8n n8n_nginx n8n_setup clamav
```

Comment:
We should take clamav out of the web commands, and jsut have it be an optional thing someone can spin up if they want to. Otherwise the "skip clamav" variable isn't doing much as we are still spending the time to spin up the service locally, only to then ignore it.

Reply:
makes sense